snap/snapcraft.yaml

```diff
@@ -102,23 +105,36 @@ apps:
       - network
       - network-bind
 parts:
+  pin-mysql-apt:
+    plugin: nil
+    override-pull: |
+      printf "Package: mysql-server\nPin: version 8.0.32*\nPin-Priority: 1001\n" > /etc/apt/preferences.d/mysql
+      printf "\nPackage: mysql-router\nPin: version 8.0.32*\nPin-Priority: 1001\n" >> /etc/apt/preferences.d/mysql
+      printf "\nPackage: percona-xtrabackup-80\nPin: version 8.0.32*\nPin-Priority: 1001\n" >> /etc/apt/preferences.d/mysql
+  package-mysql-shell:
+    plugin: dump
+    source: https://downloads.mysql.com/archives/get/p/43/file/mysql-shell-8.0.32-linux-glibc2.12-x86-64bit.tar.gz
+    organize:
+      bin: usr/bin
+      libexec: usr/libexec
+      lib: usr/lib
+      share: usr/share
   packages-deb:
     plugin: nil
     stage-packages:
       - mysql-server
```

Comment:
I recommend to switch here from mysql-server (metapackage) to mysql-server-8.0
It is minor in the moment, but more precise and can bite us on 10 years support disctance.

Reply:
Changed in [b9d9d10](https://github.com/canonical/charmed-mysql-snap/pull/23/commits/b9d9d10e3f9e9e9570e879e36da1e873935bce5b)